recipe/meta.yaml

```diff
@@ -334,3 +335,4 @@ extra:
     - erykoff
     - ktlim
     - beckermr
+    - athornton
```

Comment:
I don't know what happens if you add a maintainer to just the dev branch ...

Reply:
I don't mind taking it out.

Reply:
Done